modules/twitch-notify/events/botReady.js

```diff
@@ -0,0 +1,52 @@
+const {confDir} = require('../../../main');
+const config = require(`${confDir}/twitch-notify/config.json`);
+
+const { ApiClient } = require('twitch');
+const { ClientCredentialsAuthProvider } = require('twitch-auth');
+
+const ClientID = config['Twitch-ClientID'];
+const ClientSecret = config['ClientSecret'];
+const authProvider = new ClientCredentialsAuthProvider(ClientID, ClientSecret);
+const apiClient = new ApiClient({ authProvider });
+
+let live = false;
```

Comment:
What happens if the bot get restarted? Consider storing this and some meta data in a database and checking there if the message already got send